Source/Core/Core/HW/HSP/HSP_DeviceARAMExpansion.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/HW/HSP/HSP_DeviceARAMExpansion.h"
+
+#include "Common/ChunkFile.h"
+#include "Common/Logging/Log.h"
+#include "Common/MathUtil.h"
+#include "Common/MemoryUtil.h"
+#include "Common/Swap.h"
+
+#include "Core/Config/MainSettings.h"
+#include "Core/ConfigManager.h"
+#include "Core/HW/DSP.h"
+#include "Core/HW/HSP/HSP.h"
+#include "Core/HW/ProcessorInterface.h"
```

Comment:
These are also unused (presumably they were used by the Game Boy Player implementation?)